examples/src/main/scala/org/apache/spark/examples/mllib/NaiveBayesExample.scala

```diff
@@ -31,21 +32,17 @@ object NaiveBayesExample {
     val conf = new SparkConf().setAppName("NaiveBayesExample")
     val sc = new SparkContext(conf)
     // $example on$
-    val data = sc.textFile("data/mllib/sample_naive_bayes_data.txt")
-    val parsedData = data.map { line =>
-      val parts = line.split(',')
-      LabeledPoint(parts(0).toDouble, Vectors.dense(parts(1).split(' ').map(_.toDouble)))
-    }
+    // Load and parse the data file.
+    val data = MLUtils.loadLibSVMFile(sc, "data/mllib/sample_libsvm_data.txt")
 
     // Split data into training (60%) and test (40%).
-    val splits = parsedData.randomSplit(Array(0.6, 0.4), seed = 11L)
-    val training = splits(0)
-    val test = splits(1)
+    val splits = data.randomSplit(Array(0.6, 0.4))
+    val (trainingData, testData) = (splits(0), splits(1))
```

Comment:
Lets not change the variable names, since they were consistent.
But hey in Scala this can be `val Array(training, test) = data.randomSplit(...)`